### PR TITLE
Feature/add-credential-manager

### PR DESCRIPTION
**Pull request by majeed-platogo**

### CODE REVIEW

File: src/android/GoogleSignInPlugin.java [hunk 1]
```diff
@@ -230,79 +206,9 @@ private void respondWithGoogleToken(String idToken) {
...
-    private void saveCancelTime(ArrayList<Number> cancelTimeArray) {
-        SharedPreferences sharedPreferences = getSharedPreferences();
-        SharedPreferences.Editor preferences = sharedPreferences.edit();
-
-        String cancelTimeString = TextUtils.join(";", cancelTimeArray);
-
-        preferences.putString(Constants.PREF_CANCEL_TIME_ARRAY_STRING, cancelTimeString);
-        preferences.apply();
+    private int getAppResource() {
+        return cordova.getActivity().getResources().getIdentifier("default_client_id", "string",
```
Comment (Luro91) [inline src/android/GoogleSignInPlugin.java:210]: Not part of the PR but Android studio shows a warning here: 
>Use of this function is discouraged because resource reflection makes it harder to perform build optimizations and compile-time verification of code. It is much more efficient to retrieve resources by identifier (e.g. `R.foo.bar`) than by name (e.g. `getIdentifier("bar", "foo", null)`).

Comment (majeed-platogo) [reply src/android/GoogleSignInPlugin.java:210]: Yes, but we can't fix this because we don't know if the app is slotpark or gaminator so we can't write the class name directly

**Review by Luro91 — APPROVED** [review]
